docs/guide.md

```diff
@@ -0,0 +1,210 @@
+
+If you're starting with the example library or Yeoman, go ahead and skip to [the next section](LINK).
+
+The first step is to initialize a new project with a Git repository: create a new directory and run `git init`.
+
+### Project Dependencies
+Once your new project is created, you'll need to get a few dependencies set up:
+
+- `npm init` and follow the prompts to generate a node project
+- `npm install grunt patternpack --save-dev` to install grunt and PatternPack as dependencies
+
+At this point, we've got project dependencies ready.
+
+### Basic Configuration
+PatternPack is a grunt plugin, so you'll need to create a `gruntfile.js` in your project's root directory and add PatternPack to it:
+
+#### gruntfile.js
+```js
+module.exports = function (grunt) {
+  grunt.initConfig({
+    patternpack: {
+      run: {},
+      build: {},
+      release: {}
+    }
+  });
+
+  grunt.loadNpmTasks('patternpack');
+
+  grunt.registerTask('default', ['patternpack:run']);
+}
+```
+
+This grunt configuration exposes three tasks to your project:
+
+- `patternpack:run` which starts up the development environment
+- `patternpack:build` which generates the web server files
+- `patternpack:release`, which exposes the versioning and release tasks
+
+The last line also maps the default `grunt` command to generate your pattern library and start up a development environment.
+
+### Advanced Configuration
+PatternPack provides smart defaults, however it can be extensively configured to suit your needs. Here is what all the configurable PatternPack options look like with all options set to default:
+
+```js
+patternpack: {
+  options: {
+    release: "./dist",
+    build: "./html",
+    src: "./src",
+    assets: "./src/assets",
+    css: {
+      preprocessor: "sass",
+      fileName: "patterns",
+      autoprefixer: {
+        browsers: ["last 2 versions"]
+      }
+    }
+    integrate: "../patternpack-example-app/node_modules/patternpack-example-library",
+    theme: "./node_modules/patternpack-example-theme",
+    logo: "./theme-assets/images/logo.svg",
+    publish: {
+      library: true,
+      patterns: false
+    },
+    patternStructure: [
+      { name: "Atoms", path: "atoms" },
+      { name: "Molecules", path: "molecules" },
+      { name: "Pages", path: "pages" }
+    ],
+    server: {
+      port: 1234
+    }
+  },
+  run: {
+    task: "start"
+  },
+  build: {
+    task: "server"
+  },
+  release: {
+    task: "release"
+  }
+}
+```
+
+There are a few options you might want to customize early on based on your needs:
+
+- `options.css.fileName` - this is the "master" Sass/LESS file that will import your patterns as well as any other dependencies you want to integrate ([documentation](https://github.com/patternpack/patternpack#cssfilename))
+- `options.css.preprocessor` - defaults to Sass, but LESS is available as well ([documentation](https://github.com/patternpack/patternpack#csspreprocessor))
```

Comment:
We should promote the casing that we expect in the configuration option. Today this is all lowercase.